legistar/bills.py

```diff
@@ -236,21 +234,29 @@ def matters(self, since_datetime=None):
             except KeyError:
                 url = matters_url + '/{}'.format(matter['MatterId'])
                 self.warning('Bill could not be found in web interface: {}'.format(url))
-                continue
+                if scrape_restricted:
+                    matter['legistar_url'] = self.BASE_WEB_URL
```

Comment:
this is not the url for this item, why not leave it blank?

Reply:
We'd get a KeyError here: https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/bills.py#L154

Reply:
So, I can change the code in the Metro scraper to accommodate.

Reply:
I'd change the metro code to accomodate. You still need a conditional to avoid continuing.

I'd set the `scrape_restricted` to False on the init method of this class.

Then, i'd have a conditional like

```python
# if we don't have access to private events in the API, we want to avoid collecting them.
if not self.scrape_restricted:
    continue
```

Reply:
oh, shucks: I was just about to push this (nearly) this change.

Reply:
as my mother always says, great minds run in the same channels